saleor/graphql/app/filters.py

```diff
@@ -30,6 +31,18 @@ def filter_app_extension_mount(qs, _, value):
     return qs
 
 
+def filter_app_extension_mount_name(qs, _, value):
+    if value:
+        qs = qs.filter(mount=value.lower())
+    return qs
+
+
+def filter_app_extension_target_name(qs, _, value):
+    if value:
+        qs = qs.filter(target=value.lower())
+    return qs
```

Comment:
It may be confusing why lower()

1. Currently in DB we store enum value. Both `target`/`targetName` and `mount`/`mountName` use the same DB field, but exisitng data store lowercaise "choices", eg. `new_tab`
2. In new api user will store PLAIN `NEW_TAB`
3. At this point to make it work, lower() is needed
4. Also I will run data migration that will convert existing DB entries (from enum) to be plain text (`new_tab` →`NEW_TAB` ) - that will remove need of "lower()"
5. Finally, old fields will be dropped in 3.23, but I must ensure zero downtime transition and ensure it works on 3.22 with pre-existing installed extensions